### DISCUSSION TITLE
Why i've paid for my copilot twice??

### DISCUSSION THREAD

**Discussion by fatemagharieb** [community/community#65691] [New to GitHub]
<img width="997" alt="Screen Shot 2023-09-01 at 8 32 51 PM" src="https://github.com/community/community/assets/49796707/eae3fd6c-ef1c-4714-bb5f-c866fed1de96">

Hello all, i paid the 10 dollars for my GitHub copilot account at the 28th of August, and today the 1st of September I've got charged for another 10 dollars!

**Comment by queenofcorgis** [comment]
Hi @fatemagharieb ! The best place to go for answers to billing questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
